src/App.vue

```diff
@@ -517,12 +516,56 @@
         </div>
       </div>
     </div>
+    <div class="py-5">
+      <i-calendar
+        v-model="computedStuffingDate"
+        :disabled-previous-month="false"
+        :disabled-next-month="false"
+        :is-there-new-update="true"
+      >
+        <template #indicator>
+          <div class="flex items-center">Slot Header</div>
+        </template>
+
+        <template #content="{ isSelected, date }">
+          <div class="new-update-indicator" />
+          <div
+            v-if="isAdaData(date)"
+            class="list-ports-wrapper"
+          >
+            <div
+              v-for="(port, key) in isAdaData(date).listNya"
+              :key="key"
+              class="list-ports"
+              :class="port.isRatesAvailable ? 'tw-bg-green-400' : 'tw-bg-red-400'"
+            >
+              {{ port.origin }}-{{ port.destination }}
+            </div>
+            <div class="tw-text-start tw-text-3xs tw-font-medium">+2</div>
+          </div>
+          <i-button
+            v-if="isSelected"
+            text
+            size="xs"
+            class="btn-plan"
+          >
+            <template #prepend>
+              <ic-plus-circle class="tw-w-[11px] tw-h-[11px] tw-mr-[6px]" />
+            </template>
+            Plan
+          </i-button>
```

Comment:
![Screen Shot 2023-08-18 at 14 44 04](https://github.com/ifreight/frontend-lib/assets/54936814/299ceb8a-6384-4051-b8f4-39da109c0dd0)
![Screen Shot 2023-08-18 at 14 43 57](https://github.com/ifreight/frontend-lib/assets/54936814/b82f43aa-c1bf-4dd9-af0b-a7fcfe9ee29d)

fix the content and create example as similar as figma
- filled with content (include the pill color)
- when empty (confirm the add plan button position is stay at bottom or not)
- an example when the indicator appear

Reply:
ok